controllers.js

```diff
@@ -0,0 +1,16 @@
+const {getTopicData} = require('./models')
+const app = require("./app");
+const db = require('./db/connection');
+
+function getTopics(req, res){
+    getTopicData().then((result) => {
+        console.log(result)
+        res.status(200).send(result)
```

Comment:
You should be sending back an object with a key of topics with an array of topics. Can you think of a more semantic variable name other than result??